fastlane/Fastfile

```diff
@@ -128,8 +128,7 @@ lane :publish_release do |options|
                    )
 
   spm_github_release_url = update_spm(version: version)
-  # Firing this lane is done manually at the moment.
-  # TODO: push_pods(sync: options[:sync])
+  push_pods(sync: options[:sync])
```

Comment:
Does this also handle the pushing of the xcframework podspec?

Reply:
Yes, it does